<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инфографика</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="btn_show">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Тип диаграммы: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Название события</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="events">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>По всем событиям</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="type">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Оплаты событий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Количество участников</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Оплаты событий (по площадкам)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Количество участников (по площадкам)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btn_exit">
       <property name="styleSheet">
        <string notr="true">font: 75 pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
